conf/sidecar.yaml

```diff
@@ -23,6 +23,10 @@ cassandra_instances:
   - id: 1
     host: localhost1
     port: 9042
+    # The instance's storage directory as defined per the cassandra.storagedir property
+    # which defaults to the $CASSANDRA_HOME/data directory, but can be configured to any
+    # directory.
```

Comment:
How about also mentioning the evaluation order? The configured `storage_dir` will be ignored, if the concrete directories of different kinds are configured. For example, `data_dirs` if configured, the configured value takes the precedence when evaluating the data directories list.

Reply:
Updated.